library/common/http/header_utility.cc

```diff
@@ -9,14 +9,14 @@ namespace Envoy {
 namespace Http {
 namespace Utility {
 
-void toEnvoyHeaders(HeaderMap& transformed_headers, envoy_headers headers) {
```

Comment:
The edits to this file aren't strictly part of this diff.  I just noticed that the `toEnvoyHeaders` function has better parameter names in its `header_utility.h` file, figured I'd fix the mismatch.